tests/openscad_runner.py

```diff
@@ -121,20 +127,30 @@ class OpenScadRunner:
         set_variable_argument('$fa', 8) + set_variable_argument('$fs', 0.25)
 
     def __init__(self, file_path: Path):
-        self.openscad_binary_path = self.WINDOWS_DEFAULT_PATH
+        self.openscad_binary_path = self.find_openscad_binary()
         self.scad_file_path = file_path
         self.image_folder_base = Path('.')
         self.camera_arguments = None
         self.parameters = None
 
+    @classmethod
+    def find_openscad_binary(cls) -> Path:
+        if cls.WINDOWS_DEFAULT_PATH.exists():
+            return cls.WINDOWS_DEFAULT_PATH
+        if cls.LINUX_DEFAULT_PATH.exists():
+            return cls.LINUX_DEFAULT_PATH
```

Comment:
```suggestion
        exe= None
        if sys.platform == "windows":
            exe= cls.WINDOWS_DEFAULT_PATH
        if sys.platform == "linux":
            exe= cls.LINUX_DEFAULT_PATH
            
        if not exe or not exe.exists():
            raise OSError("openscad not found")
```

Reply:
Unfortunately, on my machine `sys.platform` returns 'win32'.  Maybe you meant to use `platform.system()`, and capitalize the first letter?

Reply:
I was debating on what you did or what's currently in the code.  I can go with your method but here's my logic:

* As I alluded to in my last comment, `sys.platform` has some gotchas, and [Python's docs](https://docs.python.org/3/library/sys.html#sys.platform) point to the `platform` module as a better choice.
* Portability and edge cases. [ `platform.system()`](https://docs.python.org/3/library/platform.html#platform.system) could return say "Darwin", and would mean Mac users can't run the test suite even if they had OpenSCAD installed in the default path.
* The current approach avoids having to import the platform module.  It's small, but is noticeable.
* Fewer number of lines, and same or fewer syscalls.

Reply:
It is the train of thought while reading the code which is a (non) issue for me. The code you wrote has a different outcome on the existence of a path. Linux devices will check if the windows path exists which is a weird check. 

The real story of this code is different behaviour when running on different platforms. So why not implement the code exactly as the story why this code should be here.

On the note of fewer sys calls, you perform 2 disk io operations on Linux and one on windows. Which is slower than a platform call. But Python is already slow so who cares. 

Regarding the platform module, your probably right. 

Regarding Darwin, it is good to actively think about if the code also needs to be compatible on Darwin as it also differs here and there. This usecase it is the same which could be a coincidence. 

This is a nit-pick from me. I'm fine eighter way, it clearly works.